### DISCUSSION TITLE
How to turn off warnings when Invoke-SqlQuery returns 0 rows?

### DISCUSSION THREAD

**Discussion by kwein123** [mithrandyr/SimplySql#144] [Q&A] [answered]
In version 2 of SimplySql, when I issue Invoke-SqlQuery such that it returns no data, I now get the polite warning that "This occurs when the query has no select statement or invokes a stored procedure that does not return a resultset."
I test for that, so it's not a problem for me.
How do I turn this warning off?

**Comment by mithrandyr** [comment, marked as answer]
The cmdlets all support common parameters.  So you can use `-WarningAction Ignore` to silence the warnings.

**Reply by kwein123** [reply]
Thanks so much!
